_component/accordion/index.md

```diff
@@ -16,32 +16,24 @@ iframe_height: medium
 
 <h3>HTML <span class="link"><a href="component.html" target="_blank">View Generated HTML</a></span></h3>
 
-<div class="code-block">
-<button class="copy-clipboard" data-clipboard-action="copy" data-clipboard-target=".copy-html-snippet">Copy to clipboard</button>
-<div class="copy-html-snippet">
-{% highlight html %}
-<script>
-	/* Add to <head> */
-	// Remove no-js and add 'js' to the HTML
-	document.documentElement.className = document.documentElement.className.replace('no-js', ' ');
-	document.documentElement.className += ' js ';
-</script>
-{% endhighlight %}
-</div>
-</div>
-
-This component requires a `js` class to be present on the `html` element to fully function. If you're using a script like [Modernizr](http://modernizr.com), it will handle this for you. If not, you'll need to add the above script to the `head` of the document. Adding it into the `head` will assure there will be no jumpiness in the UI while the JavaScript and CSS are loading.
-
-
 {% include partials/html-tabs.md %}
 
-<h3>SCSS <span class="link"><a href="scss/component.scss" target="_blank">Download SCSS</a></span></h3>
+<h3>PostCSS <span class="link"><a href="scss/component.scss" target="_blank">Download PostCSS</a></span></h3>
 {% include partials/scss-component.md %}
```

Comment:
@colorful-tones @garand This is just outputting the PostCSS, the implementation we're suggesting uses an `@import`, so let's document that instead. And we're not using scss anymore, so it we're going to use the partial, let's rename it.

Reply:
@timwright12 
Unfortunately, renaming the partial breaks the Liquid templating. So that, if I rename `scss/component.scss` to `css/component.css` then this will break this template lookup seen here: [`_includes/partials/scss-component.md`](https://github.com/10up/wp-component-library/blob/gh-pages/_includes/partials/scss-component.md). If I were to update the template then all the other components that rely on SCSS would break. 

I initially started to go down this rabbit hole, but a lot of Liquid templating would have to be refactored. I checked with @garand and it was advised it was not necessary.

Reply:
We could add a new `{% include partials/css-component.md %}`, which has similar logic as the existing `{% include partials/scss-component.md%}` in the `partials/`?

Reply:
I added a new `_includes/partials/css-component.md` and updated links.

Reply:
Adding a new partial is fine, we just can't have scss stuff lingering in areas that don't use it.